azext_connect/_params.py

```diff
@@ -19,6 +19,7 @@ def load_arguments(self, _):
         c.argument('appname', options_list=['--app-name', '-app'], help='Webapp name')
         c.argument('sql', options_list=['--sql-server', '-sql'], help='SQL server name in the same resource group or SQL server ID')
         c.argument('mysql', options_list=['--mysql-server', '-mysql'], help='SQL server name in the same resource group or mysql server ID')
+        c.argument('postgre', options_list=['--postgre-server', '-postgre'], help ='PostgreSql server name in the same resource group or PostgreSql server ID')
```

Comment:
I think "postgres" will be more accurate here.
Please keep align with the Postgres Azure CLI, it uses "az postgres ..."

Reply:
Fixed.